public_html/wp-content/themes/pattern-directory/src/components/active-filter-bar/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { select } from '@wordpress/data';
+import { store } from '@wordpress/viewport';
+import { Flex, FlexItem } from '@wordpress/components';
+
+function ActiveFilterBar( { children, actionsTitle, actions } ) {
+	const isSmall = select( store ).isViewportMatch( '< medium' );
```

Comment:
> It only works on page load right now, I'm not sure what our/gutenberg's preferred way to track window resize.

What's the reason to use a non-native solution necessary here (even one provided by the platform)? Media queries could be more straight forward and future-proof.

Reply:
Good question. 

I convinced myself to use Gutenberg's viewport hook based on these assumptions:
1. We will (probably) be using a number of Gutenberg components that will respond based on Gutenberg breakpoints. This will keep the UI in sync.
2. Using JS approach would make the component slightly more portable.

I could easily be convinced otherwise.

Reply:
I think those are good reasons, but personally I'm not sure it outweighs the resizing problems. 

There's the devex concern of course, but there's also a _UX_ concern, because some users will resize windows to fit the content, especially if they have larger monitors with room for multiple windows. 

In the Past GitHub didn't adapt when resizing the window, and it always bugged me.

That's probably a minority of users, though, so I can see the tradeoff, and don't feel strongly.

Reply:
We have the gutenberg breakpoints available in the settings, if you need them:

https://github.com/WordPress/pattern-directory/blob/3a81aeb0f200a2921b862d657134d530c82b2dd3/public_html/wp-content/themes/pattern-directory/css/settings/_structure.scss#L5-L11

Since this is just hiding content, I think a plain (CSS) media query would be better.